dbms/src/Storages/MergeTree/MergeTreeDataPartCompact.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <Storages/MergeTree/IMergeTreeDataPart.h>
+
+namespace DB
+{
+
+/// Description of the data part.
+class MergeTreeDataPartCompact : public IMergeTreeDataPart
```

Comment:
Comments. Differences between wide and compact parts. Structure on disk, structure in memory. Related settings. etc.